octoprint: 1.11.3 -> 1.11.4

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/nix-community/nixpkgs-update) tools. This update was made based on information from passthru.updateScript.

meta.description for octoprint is: Snappy web interface for your 3D printer

meta.homepage for octoprint is: https://octoprint.org/




###### Updates performed
- Ran passthru.UpdateScript

###### To inspect upstream changes

- [Release on GitHub](https://github.com/OctoPrint/OctoPrint/releases/tag/1.11.4)

- [Compare changes on GitHub](https://github.com/OctoPrint/OctoPrint/compare/1.11.3...1.11.4)

###### Impact

<b>Checks done</b>

---

- built on NixOS

> [!CAUTION]
> A test defined in `passthru.tests` did not pass.
- found 1.11.4 with grep in /nix/store/x8mi3ckbyj18gmyyhn902j6ljmghv00v-python3.13-OctoPrint-1.11.4
- found 1.11.4 in filename of file in /nix/store/x8mi3ckbyj18gmyyhn902j6ljmghv00v-python3.13-OctoPrint-1.11.4

---

<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
2 total rebuild path(s)

2 package rebuild(s)

First fifty rebuilds by attrpath

octoprint
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from the cache**:
```
nix-store -r /nix/store/x8mi3ckbyj18gmyyhn902j6ljmghv00v-python3.13-OctoPrint-1.11.4 \
  --option binary-caches 'https://cache.nixos.org/ https://nixpkgs-update-cache.nix-community.org/' \
  --option trusted-public-keys '
  nixpkgs-update-cache.nix-community.org-1:U8d6wiQecHUPJFSqHN9GSSmNkmdiFW7GW7WNAnHW0SM=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The nixpkgs-update cache is only trusted for this store-path realization.)
For the cached download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A octoprint https://github.com/r-ryantm/nixpkgs/archive/4418aa67d3d4625d72aaf5d0713dad34f77ffbd6.tar.gz
```
Or:
```
nix build github:r-ryantm/nixpkgs/4418aa67d3d4625d72aaf5d0713dad34f77ffbd6#octoprint
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/x8mi3ckbyj18gmyyhn902j6ljmghv00v-python3.13-OctoPrint-1.11.4
ls -la /nix/store/x8mi3ckbyj18gmyyhn902j6ljmghv00v-python3.13-OctoPrint-1.11.4/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review --extra-nixpkgs-config '{ allowBroken = false; }'`
Commit: `4418aa67d3d4625d72aaf5d0713dad34f77ffbd6`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 2 packages built:</summary>
  <ul>
    <li>octoprint</li>
    <li>octoprint.dist</li>
  </ul>
</details>

---

###### Maintainer pings

cc @WhittlesJr @gador for [testing](https://github.com/nix-community/nixpkgs-update/blob/main/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

> [!TIP]
> As a maintainer, if your package is located under `pkgs/by-name/*`, you can comment **`@NixOS/nixpkgs-merge-bot merge`** to automatically merge this update using the [`nixpkgs-merge-bot`](https://github.com/NixOS/nixpkgs/blob/master/ci/README.md#nixpkgs-merge-bot).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

r-ryantm committed:
octoprint: 1.11.3 -> 1.11.4

gador reviewed (dismissed):
This does not pass nixosTests due to 404 errors. This should be checked before merging

gador committed:
nixosTests.octoprint: increase delay

octoprint needs more time to start.
Increasing the retry-delay ensures the tests pass.

Signed-off-by: Florian Brandes <florian.brandes@posteo.de>
